invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -12,7 +12,10 @@
       "description": ""
     },
     "authors": {
-      "type": "string",
+      "items": {
```

Comment:
as I remember correctly we won't have the items in the schema @ntarocco @zzacharo could you guys confirm ?

Reply:
yes I agree